README.md

```diff
@@ -48,7 +48,7 @@ DATABASES = {
 $ ./manage.py migrate
 $ ./manage.py createsuperuser
 
-$ ./manage.py ingest GMLFILE
+$ ./manage.py ingest GMLFILE 160229_130000
```

Comment:
It is the format I use for the folder name of the file. I wouldn't insert a `T` for the sake of readability.